toolkit/tools/pkg/imagecustomizerlib/partitionutils.go

```diff
@@ -352,24 +405,156 @@ func findPartition(mountIdType imagecustomizerapi.MountIdentifierType, mountId s
 	return partition, partitionIndex, nil
 }
 
+func convertDevToUuid(partitions []diskutils.PartitionInfo, buildDir string) (string, error) {
+	for _, partition := range partitions {
+		matches, err := checkExtendedDevPartition(partition, partitions, buildDir)
+		if err != nil {
+			return "", err
+		}
+		if matches {
+			return partition.Uuid, nil
```

Comment:
It would probably be better to use `partition.PartUuid` here, since that is always available (for GPT disks) regardless of the filesystem being used.